draft-kazuho-httpbis-priority.md

```diff
@@ -98,6 +80,117 @@ The terms sh-token and sh-boolean are imported from
 Example HTTP requests and responses use the HTTP/2-style formatting from
 {{?RFC7540}}.
 
+# Motivation for Replacing HTTP/2 Priorities {#motivation}
+
+An important feature of any implementation of a protocol that provides
+multiplexing is the ability to prioritize the sending of information. This was
+an important realization in the design of HTTP/2. Prioritization is a
+difficult problem, so it will always be suboptimal, particularly if one endpoint
+operates in ignorance of the needs of its peer.
+
+HTTP/2 introduced a complex prioritization signaling scheme that used a
+combination of dependencies and weights, formed into an unbalanced tree. This
+scheme suffers from poor deployment and interoperability.
+
+The rich flexibility permitted in tree building is rarely excersised by clients;
+they generally either chosen a model optimized for a web use case or do nothing
+at all.
+
+Many server implementations do not include support for this scheme, some
+favoring instead bespoke server-driven schemes based on heuristics and other
+hints, like the content type of resources and the order in which requests
+arrive. For example, a server, with knowledge of the document structure, might
+want to prioritize the delivery of images that are critical to user experience
+above other images, but below the CSS files. But with the HTTP/2 prioritization
+scheme, it is impossible for the server to determine how such images should be
+prioritized against other responses that use the client-driven prioritization
+tree, because every client builds the HTTP/2 prioritization tree in a different
+way.
+
+Intermediaries can coalesce multiple HTTP/2 clients into a single upstream
+HTTP/2 connection and the prioritization scheme allows for this. However most
+intermediaries do not support this and it does not define a method that can be
```

Comment:
Maybe at least split the sentence, as the topic seems to change? The first half of this paragraph (up to "do not support this") talks about passing the tree through intermediaries, while the second half discusses about the incapability of the prioritization scheme not allowing the server to express their priorities.

Reply:
oops I messed this up a bit, PTAL at https://github.com/kazuho/draft-kazuho-httpbis-priority/pull/61/commits/7cfe822c09cfd9457d0acc95a1696d58a7cc1628